diff --git a/node_modules/tsdav/dist/tsdav.cjs.js b/node_modules/tsdav/dist/tsdav.cjs.js
index ab716b1..21112e2 100644
--- a/node_modules/tsdav/dist/tsdav.cjs.js
+++ b/node_modules/tsdav/dist/tsdav.cjs.js
@@ -232,6 +232,9 @@ const davRequest = (params) => __awaiter(void 0, void 0, void 0, function* () {
         },
         ignoreDeclaration: true,
     });
+    if (typeof result.multistatus.response === 'undefined') {
+      return [];
+    }
     const responseBodies = Array.isArray(result.multistatus.response)
         ? result.multistatus.response
         : [result.multistatus.response];
